content/docs/setup/kubernetes/additional-setup/sidecar-injection/index.md

```diff
@@ -160,20 +160,33 @@ sleep-776b7bcdcd-gmvnr   1/1       Running       0          2s
 
 #### Understanding what happened
 
-[admissionregistration.k8s.io/v1beta1#MutatingWebhookConfiguration](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#mutatingwebhookconfiguration-v1beta1-admissionregistration-k8s-io)
-configures when the webhook is invoked by Kubernetes. The default
-supplied with Istio selects pods in namespaces with label
-`istio-injection=enabled`.  The set of namespaces in which injection
-is applied can be changed by editing the `MutatingWebhookConfiguration`
-with `kubectl edit mutatingwebhookconfiguration
-istio-sidecar-injector`.
+When Kubernetes invokes the webhook, the [admissionregistration.k8s.io/v1beta1#MutatingWebhookConfiguration](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#mutatingwebhookconfiguration-v1beta1-admissionregistration-k8s-io)
+
+configuration is applied. The default configuration labels the pods in the
```

Comment:
The default configuration injects the sidecar into pods in any namespace with a `istio-injection=enabled` label. The istio-sidecar-injector ConfigMap specifies the configuration for the injected sidecar. 
To change how namespaces are selected for injection, you can edit the `MutatingWebhookConfiguration` with...